docs/introduction/overview/index.html

```diff
@@ -69,4 +69,4 @@ <h3><a class="anchor" aria-hidden="true" id="auspice-is-software-to-display-beau
-</span></div></article></div><div class="docLastUpdate"><em>Last updated on 3/24/2020</em></div><div class="docs-prevnext"><a class="docs-next button" href="/auspice/introduction/install"><span>Install Auspice</span><span class="arrow-next"> →</span></a></div></div></div><nav class="onPageNav"><ul class="toc-headings"><li><a href="#license-and-copyright">License and Copyright</a></li></ul></nav></div><footer class="nav-footer" id="footer"><section class="sitemap"><div><a href="/auspice/"><img style="padding-left:20px" src="/auspice/img/logo-light.svg" alt="Auspice" width="66" height="58"/></a></div><div><h5>External Links</h5><a href="https://github.com/nextstrain/auspice">GitHub repo</a><a href="https://www.npmjs.com/package/auspice">NPM package</a><a href="https://nextstrain.org">Nextstrain</a></div><div><h5>Contact Us</h5><a href="mailto:hello@nextstrain.org">email</a><a href="https://twitter.com/hamesjadfield">twitter</a></div></section><section class="copyright">Website built by <a href="https://twitter.com/hamesjadfield">James Hadfield</a> using <a href="https://docusaurus.io">Docusaurus</a></section><section class="copyright">If you use auspice, please cite <a href="https://doi.org/10.1093/bioinformatics/bty407">Hadfield et al., 2018</a></section><section class="copyright">Copyright © 2014-2020 Richard Neher &amp; Trevor Bedford</section></footer></div></body></html>
\ No newline at end of file
+</span></div></article></div><div class="docLastUpdate"><em>Last updated on 24/03/2020</em></div><div class="docs-prevnext"><a class="docs-next button" href="/auspice/introduction/install"><span>Install Auspice</span><span class="arrow-next"> →</span></a></div></div></div><nav class="onPageNav"><ul class="toc-headings"><li><a href="#license-and-copyright">License and Copyright</a></li></ul></nav></div><footer class="nav-footer" id="footer"><section class="sitemap"><div><a href="/auspice/"><img style="padding-left:20px" src="/auspice/img/logo-light.svg" alt="Auspice" width="66" height="58"/></a></div><div><h5>External Links</h5><a href="https://github.com/nextstrain/auspice">GitHub repo</a><a href="https://www.npmjs.com/package/auspice">NPM package</a><a href="https://nextstrain.org">Nextstrain</a></div><div><h5>Contact Us</h5><a href="mailto:hello@nextstrain.org">email</a><a href="https://twitter.com/hamesjadfield">twitter</a></div></section><section class="copyright">Website built by <a href="https://twitter.com/hamesjadfield">James Hadfield</a> using <a href="https://docusaurus.io">Docusaurus</a></section><section class="copyright">If you use auspice, please cite <a href="https://doi.org/10.1093/bioinformatics/bty407">Hadfield et al., 2018</a></section><section class="copyright">Copyright © 2014-2020 Richard Neher &amp; Trevor Bedford</section></footer></div></body></html>
```

Comment:
> @HashTalmiz this line (and many others) that are supposedly changed in the PR have the exact same content as in master, which makes me think that something (perhaps the editor you're using) is adding different line ending characters (or something else invisible like tabs vs spaces) to all the lines that you edit
> 
@eharkins I think you are referring to things like this? I almost agreed with you there but then I realized, every single one of these **Lookalikes** have the **Date format changed!**
For instance, look here, at the start of the second line on both sides
`3/24/2020` on the left and `24/03/2020` on the right
I'm not sure how this came to be. I'm 100% sure I haven't opened these files for my editor to do this. Should I remove these from the PR right from github?

Reply:
This change is [from master](https://github.com/nextstrain/auspice/commit/92659312878d0bae839900feaa93853df4dd315a#diff-3bd4770b7a8ae6e7c5940208f9faabc8R126)